src/styles/card.css

```diff
@@ -0,0 +1,99 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  margin: 0;
```

Comment:
To prevent resetting margins styles every time, reset them in the `reset.css` using tag names

Reply:
You should implement it everywhere, not only here